crates/re_viewer/src/misc/caches/tensor_decode_cache.rs

```diff
@@ -0,0 +1,135 @@
+use re_log_types::component_types::{Tensor, TensorDimension, TensorId, TensorTrait};
+
+#[derive(thiserror::Error, Clone, Debug)]
+pub enum TensorDecodeError {
+    // TODO(jleibs): It would be nice to just transparently wrap
+    // `image::ImageError` and `tensor::TensorImageError` but neither implements
+    // `Clone`, which we need if we want to cache the Result.
```

Comment:
Wrap them in `Clone` then? Or at least include their messages as `String`. Just saying `Failed to decode` without any further info is not great.